### ISSUE TITLE
Inconsistent treatment of trailing whitespace in multiline strings

### ISSUE THREAD

**Issue by daveware-nv** [PyCQA/pylint#3822]
### Steps to reproduce
```python
def example(*args):
    print(*args)


example(
    "bob", """
    foobar 
    more text
""",
)

example(
    "bob",
    """
    foobar2 
    more text
""",
)
```

### Current behavior
pylint incorrectly warns about the trailing whitespace on the line with `foobar2` and correctly ignores the trailing whitespace on the line with `foobar`.

### Expected behavior
pylint should produce no trailing whitespace warnings, as all the occurrences are in literal strings.

### pylint --version output
pylint 2.6.0
astroid 2.4.2
Python 3.7.8 (default, Jul 30 2020, 09:48:41) 
[GCC 9.3.0]

**Comment by Pierre-Sassoulas** [comment]
I can reproduce, this is very surprising.
